docs/core/quickstart-from-source.md

```diff
@@ -8,29 +8,38 @@ For a start which requires only the JRE have a look [here](../web/quickstart.md)
 Windows user can find a quick guide [here](./windows-setup.md).
 People with IDE knowledge can directly jump to [Start Development](#start-development)
 
-Now, before you proceed install git and jdk8, then do:
+Now, before you proceed install git and jdk8 or jdk11, then do:
```

Comment:
Maybe it would make sense to split this in two lines. It's too easy to misread this sentence IMHO :).

Install git and jdk8 or jdk11.

Then do: